src/mmw/js/src/core/views.js

```diff
@@ -346,13 +340,29 @@ var MapView = Marionette.ItemView.extend({
         map.addLayer(this._dataCatalogActiveLayer);
     },
 
-    setupGeoLocation: function(maxAge) {
+    fitToDefaultBounds: function() {
+        // Center the map on the U.S.
+        this._leafletMap.fitBounds([
+            [24.2, -126.4],
+            [49.8, -66.0]
+        ]);
+    },
+
+    /**
+    Attempt to locate the user and set the map to it
+    @param alwaysSetView -- override the map model setting `geolocationEnabled` (used to prevent
```

Comment:
I like this docstring style, so I'm in favor of keeping it even though the other method comments in this file are more terse.